src/url-handling.js

```diff
@@ -13,29 +13,21 @@ import {
   parseTransforms,
 } from './profile-logic/transforms';
 import { assertExhaustiveCheck, toValidTabSlug } from './utils/flow';
+import * as AppActions from './actions/app';
+import {
+  getUrlState,
+  getSelectedTab,
+  getDataSource,
+} from './reducers/url-state';
+import shallowEqual from 'shallowequal';
+import { sendAnalytics } from './utils/analytics';
+
+import type { Store } from './types/store';
 import type { UrlState } from './types/reducers';
 import type { DataSource } from './types/actions';
 
 export const CURRENT_URL_VERSION = 2;
 
-function dataSourceDirs(urlState: UrlState) {
```

Comment:
This is moved below, as this file is larger now and I wanted it to read better in how the files were organized.